eventually/authentication/models.py

```diff
@@ -0,0 +1,161 @@
+from django.db import models
+
+# Create your models here.
```

Comment:
We don't need commented line in git repository

Reply:
Deleted.